### DISCUSSION TITLE
Chat using gupshup whatsapp API, signalr service and aspnet MVC

### DISCUSSION THREAD

**Discussion by ishu58** [ColorlibHQ/AdminLTE#3315] [Q&A] [answered]
hello sir /madam,

 actually, I want to create WhatsApp web-like functionality in my  CRM project. I have a Gupshup API  but it asks for webhooks (call back URL ).

Could please suggest how to create webhooks and how to implement in my mvc5 project using signalR for real-time chatting like WhatsApp?

I want to integrate Gupshup  Whatsapp API using signalR in asp.net MVC 5

**Comment by REJack** [comment, marked as answer]
For that you should ask this questions on stackoverflow or something else. That more a programming questions than a question related to AdminLTE itself.
